webservices/utils.py

```diff
@@ -107,7 +108,7 @@ def fetch_page(
             index_column=index_column,
             nulls_last=nulls_last,
         )
-    paginator = paginators.OffsetPaginator(query, kwargs["per_page"], count=count)
+    paginator = paginators.OffsetPaginator(query, kwargs["per_page"], session, count=count)
```

Comment:
I'm going to look into binding the session so hopefully we won't have to pass it in!